source/_integrations/oru.markdown

```diff
@@ -0,0 +1,32 @@
+---
+title: "Orange and Rockland Utility Real Time Energy Usage Sensor"
+description: "Instructions on how to integrate the Orange and Rockland Utility real time energy usage sensor within Home Assistant."
+logo: oru.png
```

Comment:
@klaasnicolaas Any idea where the logo image file should be hosted? Thanks.

Reply:
@bvlaicu It should be in the folder: `source/images/supported_brands`